build.sbt

```diff
@@ -26,6 +26,8 @@ scalaVersion := "2.12.12"
 
 resolvers += Resolver.bintrayRepo("hmrc", "releases")
 
+Test / fork := false
```

Comment:
This is to disable forking for testing. Without this tests had some issues around Futures.

Reference: https://www.scala-sbt.org/1.0/docs/Forking.html